### PR TITLE
Created new percent active graph and display active, inactive, and total number of users 

### PR DESCRIPTION
**Pull request by rickypchen**
Addressed changes proposed in google doc: https://docs.google.com/document/d/1DrXuXIXVCDfvEaDRoBX6klct7FP9EaNRcyl3Bc6qeTc/edit#

Highlights:
1. categories changed to high performing and low performing
2. new active users chart (% of active users) and tooltip displaying # of active/inactive/total users 

pinging @benrudolph for code review

### CODE REVIEW

File: corehq/apps/hqwebapp/templatetags/hq_shared_tags.py [hunk 1]
```diff
@@ -80,8 +80,8 @@ def add_days(date, days=1):
     try:
         return date + span
     except:
-        return datetime.strptime(date,'%m/%d/%Y').date() + span 
-    
+        return datetime.strptime(date,'%m/%d/%Y').date() + span
```
Comment (dimagimon) [inline corehq/apps/hqwebapp/templatetags/hq_shared_tags.py]: E231 missing whitespace after ','

File: corehq/apps/reports/templates/reports/project_health/partials/user_list.html [hunk 2]
```diff
@@ -3,9 +3,13 @@
 <table class="table table-striped">
     <thead>
         <tr>
+            <th></th>
+            <th colspan="2">{% trans "Change in Forms Submitted" %}</th>
+        </tr>
```
Comment (benrudolph) [inline corehq/apps/reports/templates/reports/project_health/partials/user_list.html]: confirming that this is supposed to be two header rows (normally there's just one)

File: corehq/apps/reports/templates/reports/project_health/project_health_dashboard.html [hunk 3]
```diff
@@ -49,14 +56,73 @@
...
+            chart.color(["#004ebc", "#e53e30"]);
+            d3.select('#perform_chart svg')
                 .datum([performingSeries, activeNonPerformingSeries])
                 .call(chart);
             nv.utils.windowResize(chart.update);
             return chart;
         });
     }
 
+    function parseToUTCDate(month){
```
Comment (benrudolph) [inline corehq/apps/reports/templates/reports/project_health/project_health_dashboard.html]: how does this parse a month to a UTC date? what exactly is being passed in? we use moment.js on a few of our pages which is very helpful for working with dates:http://momentjs.com/

**Comment by benrudolph** [comment]
👍 looking good! just a few minor comments

File: corehq/apps/reports/templates/reports/project_health/partials/user_list.html (hunk 2 above)
Comment (rickypchen) [inline corehq/apps/reports/templates/reports/project_health/partials/user_list.html]: @benrudolph Yup, I think this was the layout that Sheel had in mind per the google doc: https://docs.google.com/document/d/1DrXuXIXVCDfvEaDRoBX6klct7FP9EaNRcyl3Bc6qeTc/edit#

File: corehq/apps/reports/templates/reports/project_health/project_health_dashboard.html (hunk 3 above)
Comment (rickypchen) [inline corehq/apps/reports/templates/reports/project_health/project_health_dashboard.html]: @benrudolph Oh cool, I can definitely take a look at momentjs! I was working with the JS Date objects and dealing with timezone issues, but momenjs will be super helpful

File: corehq/apps/reports/templates/reports/project_health/partials/user_list.html (hunk 2 above)
Comment (benrudolph) [inline corehq/apps/reports/templates/reports/project_health/partials/user_list.html]: 👍

File: corehq/apps/reports/templates/reports/project_health/project_health_dashboard.html [hunk 4]
```diff
@@ -19,6 +19,7 @@
     </style>
 {% endblock %}
 {% block js %}{{ block.super }}
+    <script src="{% static 'moment/moment.js' %}"></script>
```
Comment (benrudolph) [inline corehq/apps/reports/templates/reports/project_health/project_health_dashboard.html]: https://github.com/dimagi/commcare-hq/blob/master/corehq/apps/style/templates/style/base.html#L187

i think we already include this library by default

**Pull request merged**